src/main/java/Event.java

```diff
@@ -0,0 +1,8 @@
+public class Event extends Task {
+
+    public Event(String description) {
+        super(description);
+        this.taskType = "[E]";
```

Comment:
taskType is a bit vague without the context of the question. Perhaps a comment could be added her to explain a bit